### PR TITLE
Rename project to adc-streaming

### LINKED ISSUE #15: Rename genesis-client to `adc-streaming`
`genesis-client` is a rather non-descriptive name for what this library is evolving to -- a light-weight transport-protocol agnostic interface to pub/sub systems commonly used in astronomy. I think it therefore deserves a new name, and I'd propose 'neutrino'.

Pros:
* [ ] Because, just like our library, neutrinos are lightweight and fast
* [ ] Because it will be easier to remember than genesis-client
* [ ] Because the tagline could be "neutrino - [free streaming](https://en.wikipedia.org/wiki/Free_streaming)", which makes the inner physicist in me very happy :-)
* [ ] `neutrino` is available on `conda-forge`
* [ ] Because it _might_ be possible to get [neutrino on PyPI](https://pypi.org/project/neutrino/), as it seems abandoned.

Cons:
* [ ] `neutrino` is not descriptive of the underlying functionality. Reminds me how OpenStack has all weirdly named projects (nova, swift, keystone, neutron, ...) and I can never remember which one is which.
* [ ] We may not get the PyPI name, and would need to come up with something like `neutrino-lib` or similar.
* [ ] Google-ability of just "neutrino" is not ideal.

### PR DESCRIPTION
**Pull request by spenczar**
This renames things so that you `pip install adc-streaming`, and you `import adc.streaming`. Once merged, I will rename the repository in Github too. I think we will march versioning forward - we won't rewind to 0.0.1 or anything.

Fixes #15.

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -22,9 +22,9 @@
 
 # -- Project information -----------------------------------------------------
 
-project = u'genesis-client'
-copyright = u'2019, Mario Juric'
-author = u'Mario Juric'
+project = u'adc-streaming'
+copyright = u'2020, Astronomy Data Commons Team'
+author = u'Astronomy Data Commons Team'
```
Comment (spenczar) [inline doc/conf.py:27]: Let me know if this should be something different.

File: setup.py [hunk 2]
```diff
@@ -85,23 +85,23 @@ def git_version():
...
           "Development Status :: 3 - Alpha",
           "Operating System :: POSIX :: Linux",
           "Operating System :: MacOS :: MacOS X"
       ],
-      author='Mario Juric',
-      author_email='mjuric@astro.washington.edu',
-      license='MIT',
-      packages=['genesis'],
+      author='Astronomy Data Commons Team',
+      author_email='swnelson@uw.edu',
```
Comment (spenczar) [inline setup.py:102]: This one too - let me know if it should be different.

File: doc/conf.py (hunk 1 above)
Comment (mjuric) [reply doc/conf.py:27]: LGTM

File: setup.py (hunk 2 above)
Comment (mjuric) [reply setup.py:102]: Looks good

**Review by mjuric — APPROVED** [review]
All looks good, cool!

Remind us we'll need to store the credentials for `adc-streaming` (and `adc`) on PyPI somewhere so we both have access to it in case of emergency.

**Comment by spenczar** [comment]
I'm actually unable to rename this repo, because I don't have permission to.

@mjuric, can you either rename the repo, or give me admin permissions over this repository? (https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-an-individuals-access-to-an-organization-repository)

**Comment by mjuric** [comment]
@spenczar I made you the owner of the organization -- feel free to change names as needed.

**Comment by spenczar** [comment]
Uploaded to PyPI: https://pypi.org/project/adc-streaming/
Uploaded to Anaconda: https://anaconda.org/swnelson/adc-streaming

**Comment by spenczar** [comment]
Duh, that's my personal anaconda, rather than scimma. Let's talk about getting access offline.

**Comment by mjuric** [comment]
@spenczar You're now a co-owner of the `astronomy-commons` and `scimma` anaconda.org organizations; you should be able to upload (and manage the orgs as necessary).

History: adc should properly live in `astronomy-commons`, but I also uploaded copies to `scimma` to make it possible to `conda install` scimma packages w/o having to specify the `astronomy-commons` channel. This is a user-friendly hack until we got `adc-streaming` (nee `genesis-client`) a new name (✅), proper packaging (✅), and added it to [conda-forge](https://conda-forge.org/) (TODO). May be good to open an issue for the latter.

**Pull request merged**